styles.css

```diff
@@ -0,0 +1,197 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(199, 199, 199);
+  font-family: 'Crete Round', 'Inter', 'Lato', sans-serif;
```

Comment:
Use generic font instead of regular font or include a link within the head section.